src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books';
+import categoryReducer from './categories';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    categories: categoryReducer,
+  },
+});
+
```

Comment:
- [ ] You have done a great job on this milestone but there are still some issues you still need to address, According to the project requirement kindly Structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. Your files/folder structure could look like this:
- [ ] see project requirement screenshot below
![22 08 2023_10 12 21_REC](https://github.com/noelfoka/bookstore/assets/92458236/d3d46db3-434f-49fa-8336-6dcb87a1f49f)